### ISSUE TITLE
Touchpad scrolling crashes game

### ISSUE THREAD

**Issue by t-karcher** [unknown-horizons/unknown-horizons#2909]
Using two-finger scrolling on my touchpad immediately crashes the game, even in the start menu. Mouse wheel scrolling works as expected (does nothing in the menu, scrolls just fine in the game).

**Steps to reproduce:**
Start game, scroll anywhere (up, down, left, right) using the two-finger gesture on the touchpad. Game crashes.

Python version: sys.version_info(major=3, minor=7, micro=0, releaselevel='final', serial=0)
Platform: Windows-10-10.0.17134-SP0
Using fife version 0.4.2, at least 0.4.2 required
(Unknown-Horizons-2019.1.214-Setup-VC15-x86.exe)

**Comment by jmdejong** [comment]
This looks similar to https://github.com/unknown-horizons/unknown-horizons/issues/2753 which I thought I fixed before (in https://github.com/unknown-horizons/unknown-horizons/pull/2829, which is in 2019.1).
Does this indeed happen with any horizontal scrolling?

**Comment by t-karcher** [comment]
Yes, moving two fingers anywhere in any game state leads to an immediate crash. I first came across this when I tried zooming in the map with the touchpad zoom gesture, but it also crashes in the menu directly after startup when moving two fingers in any direction on the touchpad.

**Comment by jmdejong** [comment]
ok, my guess now is that it's not related to the previous scrolling issues, but it is a problem in FIFE in handling the events.

The latest version of SDL has the SDL_FINGERMOTION event type, but [fife's eventmanager](https://github.com/fifengine/fifengine/blob/e876e86d09099e6f8a55d40c3db0ad4550c29c69/engine/core/eventchannel/eventmanager.cpp#L343) has a switch statement that does not include that event or a default case

**Comment by jakoch** [comment]
This is a FIFE issue. 
The Eventhandling needs to be improved to support touchscreens:
  - SDL2 - SDL_TouchFingerEvent
  - SDL3, https://github.com/libsdl-org/SDL/blob/main/include/SDL3/SDL_touch.h
